webpack.config.js

```diff
@@ -1,9 +1,57 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const path = require('path');
+
+const extractCSS = new ExtractTextPlugin('[name].css');
+const extractSASS = new ExtractTextPlugin('[name].css');
```

Comment:
we probably only need sass plugin here, and should rewrite existing css into sass.